go.mod

```diff
@@ -11,4 +11,6 @@ require (
 	gopkg.in/macaroon.v2 v2.1.0
 )
 
+replace github.com/lightningnetwork/lnd => github.com/orbitalturtle/lnd v0.0.1-beta
```

Comment:
Can remove the replace directive now and reference the merge commit of lnd above.

Reply:
Hmmm so if I use go get github.com/lightningnetwork/lnd@5f43ef5816104f0875460ff24c697f115d97fc69 on the baking macaroon commit from #5304, I get this error:

```
github.com/lightningnetwork/lnd/lncfg
# github.com/lightningnetwork/lnd/lncfg
../../../go/pkg/mod/github.com/lightningnetwork/lnd@v0.13.0-beta.rc5.0.20210914212831-449f84d849bc/lncfg/db.go:93:43: too many arguments in call to kvdb.StartEtcdTestBackend
../../../go/pkg/mod/github.com/lightningnetwork/lnd@v0.13.0-beta.rc5.0.20210914212831-449f84d849bc/lncfg/db.go:95:37: db.Etcd.EmbeddedLogFile undefined (type *etcd.Config has no field or method EmbeddedLogFile)
../../../go/pkg/mod/github.com/lightningnetwork/lnd@v0.13.0-beta.rc5.0.20210914212831-449f84d849bc/lncfg/db.go:190:11: db.Etcd.CloneWithSubNamespace undefined (type *etcd.Config has no field or method CloneWithSubNamespace)
../../../go/pkg/mod/github.com/lightningnetwork/lnd@v0.13.0-beta.rc5.0.20210914212831-449f84d849bc/lncfg/db.go:199:11: db.Etcd.CloneWithSubNamespace undefined (type *etcd.Config has no field or method CloneWithSubNamespace)
../../../go/pkg/mod/github.com/lightningnetwork/lnd@v0.13.0-beta.rc5.0.20210914212831-449f84d849bc/lncfg/db.go:209:11: db.Etcd.CloneWithSubNamespace undefined (type *etcd.Config has no field or method CloneWithSubNamespace)
../../../go/pkg/mod/github.com/lightningnetwork/lnd@v0.13.0-beta.rc5.0.20210914212831-449f84d849bc/lncfg/db.go:219:11: db.Etcd.CloneWithSubNamespace undefined (type *etcd.Config has no field or method CloneWithSubNamespace)
../../../go/pkg/mod/github.com/lightningnetwork/lnd@v0.13.0-beta.rc5.0.20210914212831-449f84d849bc/lncfg/db.go:229:11: db.Etcd.CloneWithSubNamespace undefined (type *etcd.Config has no field or method CloneWithSubNamespace)
../../../go/pkg/mod/github.com/lightningnetwork/lnd@v0.13.0-beta.rc5.0.20210914212831-449f84d849bc/lncfg/db.go:239:11: db.Etcd.CloneWithSubNamespace undefined (type *etcd.Config has no field or method CloneWithSubNamespace)
```

But if I go get a more recent commit the error disappears. Any reason not to use the more recent commit?

Reply:
Ah yes, there was a problem with the `kvdb` submodule that has been fixed in the meantime. No reason against using the most recent commit from `master` :+1: